src/components/test/QuickOpenModal.spec.js

```diff
@@ -367,7 +367,7 @@ describe("QuickOpenModal", () => {
       };
       wrapper.find("SearchInput").simulate("keydown", event);
       expect(props.setQuickOpenQuery).not.toHaveBeenCalled();
-      expect(props.selectLocation).not.toHaveBeenCalled();
+      expect(props.selectSpecificLocation).not.toHaveBeenCalled();
```

Comment:
all this `selectSpecificLocation` is here so that in some places we can say, YES, we really do want to go somewhere